docs/docs-book/src/using-coffee.md

```diff
@@ -129,6 +129,14 @@ or to see all the plugins inside the repositories
 coffee list --remotes
 ```
 
+## Searching for a plugin
+
+> ✅ Implemented
+
+```bash
+coffee search <NAME_OF_PLUGIN>
```

Comment:
```suggestion
coffee search <plugin_name>
```

Do not use capitalize letter

Reply:
should I change all the docs to use plugin_name instead of NAME_OF_PLUGIN?

Reply:
yes but not in this PR